api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/controllers/api/StaticAPIRefreshController.java

```diff
@@ -18,18 +18,18 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-import org.zowe.apiml.apicatalog.staticapi.StaticAPIService;
+import org.zowe.apiml.apicatalog.staticapi.StaticService;
 import reactor.core.publisher.Mono;
 import reactor.core.scheduler.Schedulers;
 
 @RequiredArgsConstructor
 public class StaticAPIRefreshController {
 
-    private final StaticAPIService staticAPIService;
+    private final StaticService staticService;
 
     @PostMapping(value = "/refresh", produces = MediaType.APPLICATION_JSON_VALUE)
     public Mono<ResponseEntity<String>> refreshStaticApis() {
-        return Mono.fromCallable(staticAPIService::refresh)
+        return Mono.fromCallable(staticService::refresh)
```

Comment:
are you sure this call still works? if it calls the function directly it may be using blocking code to read the files.

Reply:
It may need the publishOn() option.
Did you try this call in a running application?

Reply:
I guess to, it is calling `.subscribeOn(Schedulers.boundedElastic())` at org.zowe.apiml.apicatalog.controllers.api.StaticAPIRefreshController#refreshStaticApis